src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanJavaAligner.java

```diff
@@ -69,113 +47,35 @@ public Parameters(final int w_match, final int w_mismatch, final int w_open, fin
         CLIP
     }
 
-    /**
-     * What strategy should we use when the best path does not start/end at the corners of the matrix?
-     */
-    public enum OverhangStrategy {
-        /*
-         * Add softclips for the overhangs
-         */
-        SOFTCLIP,
-
-        /*
-         * Treat the overhangs as proper insertions/deletions
-         */
-        INDEL,
-
-        /*
-         * Treat the overhangs as proper insertions/deletions for leading (but not trailing) overhangs.
-         * This is useful e.g. when we want to merge dangling tails in an assembly graph: because we don't
-         * expect the dangling tail to reach the end of the reference path we are okay ignoring trailing
-         * deletions - but leading indels are still very much relevant.
-         */
-        LEADING_INDEL,
-
-        /*
-         * Just ignore the overhangs
-         */
-        IGNORE
-    }
-
-    private SWPairwiseAlignmentResult alignmentResult;
-
-    private final Parameters parameters;
-
     private static final boolean cutoff = false;
```

Comment:
Any idea what the intention here is ? This is private and appears to be read-only. If toggled would affect every consumer of the singleton instance. Seems like the dead code paths should be removed.

Reply:
yeah, it's gross, I was torn between desire to remove it and desire to leave things as they were.  I'll kill it.